trymito.io/components/Glossary/HorizontalNav/HorizontalNav.module.css

```diff
@@ -0,0 +1,9 @@
+.container {
```

Comment:
Although, maybe we don't have to, because they aren't global styles? Even then, I think good practice to make these more specific.

Reply:
I'm going to leave these for now because that's what we use across this codebase.